server/modules/record/manager/recordManager.js

```diff
@@ -49,7 +49,13 @@ const fetchRecordAndNodesByUuid = async (surveyId, recordUuid, draft = true, cli
   const record = await fetchRecordByUuid(surveyId, recordUuid, client)
   const nodes = await NodeRepository.fetchNodesByRecordUuid(surveyId, recordUuid, draft, client)
 
-  return Record.assocNodes(SurveyUtils.toUuidIndexedObj(nodes))(record)
+  return Record.assocNodes(ObjectUtils.toUuidIndexedObj(nodes))(record)
+}
+
+//DELETE
+const deleteNodes = async (surveyId, nodeDefsUuids, record, client = db) => {
```

Comment:
update and delete methods are exposed through RecordUpdateManager. please move it there.
Also, what about calling this method deleteNodesByNodeDefUuids?

Reply:
ok